### DISCUSSION TITLE
AP2 and apple TV 4K feedback

### DISCUSSION THREAD

**Discussion by NicDesrochers** [mikebrady/shairport-sync#1344] [General]
Hi Mike,

Thank's for this very good projet!
I use shairport-sync since many years and now testing the dev branch for AP2
Work really good I run it from Raspberry PI4 with HiFIBerry AMP2 hat (sndrpihifiberry in alsa)

I test it with my apple TV 4K as airplay speaker and it's work great in AP2 with sound sync

pi@raspi:~/dev/shairport-sync $ ./shairport-sync -V
4.1-dev-22-gc3102201-AirPlay2-alac-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc

./configure --sysconfdir=/etc --with-alsa \
--with-soxr --with-avahi --with-ssl=openssl --with-systemd --with-airplay-2 \
--with-apple-alac

I also try with the alac flag but I do not know if it's now working or using AAC codec as writen in the readme

**Comment by mikebrady** [comment]
Many thanks, and apologies for the delay!
